stack/rush-stack-compiler/src/ApiExtractorRunner.ts

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import {
+  FileSystem,
+  ITerminalProvider
+} from '@microsoft/node-core-library';
+import {
+  Extractor,
+  IExtractorOptions,
+  IExtractorConfig
+} from '@microsoft/api-extractor';
+
+import { Constants } from './Constants';
+import { RushStackCompilerBase } from './RushStackCompilerBase';
+import { ToolPaths } from './ToolPaths';
+
+/**
+ * @public
+ */
+export interface IApiExtractorTaskConfig {
```

Comment:
>IApiExtractorTaskConfig [](start = 17, length = 23)

 This is the GCB wrapper for **api-extractor.json**,  I really want to eliminate this.

Reply:
@sunilsurana is working on implementing `extends` support for **api-extractor.json**.  Let's wait for that to be completed, and then we'll eliminate this wrapper.

---
In reply to: [228679735](https://github.com/Microsoft/web-build-tools/pull/905#discussion_r228679735) [](ancestors = 228679735)